tutorials/internal/llama_index_tracing_example.ipynb

```diff
@@ -76,11 +75,7 @@
     "if not (openai_api_key := os.getenv(\"OPENAI_API_KEY\")):\n",
     "    openai_api_key = getpass(\"🔑 Enter your OpenAI API key: \")\n",
     "openai.api_key = openai_api_key\n",
-    "os.environ[\"OPENAI_API_KEY\"] = openai_api_key\n",
-    "\n",
-    "if not (cohere_api_key := os.getenv(\"COHERE_API_KEY\")):\n",
```

Comment:
this is the internal one - it seemed nice to have a notebook that executed a reranker?

Reply:
oh, my bad. I had mistaken this one with the public one `llama_index_tracing_tutorial.ipynb` which has a very similar name.

Reply:
reverted